packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -952,17 +962,58 @@ const finish = ({ state, facets: { helper } }) => {
   });
 };
 
-const behavior = {
-  collateral: collateralBehavior,
-  helper: helperBehavior,
-  manager: managerBehavior,
-  self: selfBehavior,
-};
-
-const makeVaultManagerKit = defineDurableKindMulti(
+const makeVaultManagerKit = defineDurableFarClassKit(
   makeKindHandle('VaultManagerKit'),
+  {
+    collateral: M.interface('collateral', {
+      makeVaultInvitation: M.call().returns(InvitationShape),
+      getSubscriber: M.call().returns(SubscriberShape),
+      getMetrics: M.call().returns(SubscriberShape),
+      getCompoundedInterest: M.call().returns(RatioShape),
+    }),
+    helper: M.interface(
+      'helper',
+      // not exposed so sloppy okay
```

Comment:
It sorta feels like I should back up and double-check that this is not exposed. Not just now...